data/dev/attribute.csv

```diff
@@ -10,7 +10,9 @@ id,attribute_name,definition,model,structured_comment_name,value_syntax,allowed_
 318,sample source,"additional information about where the sample came from, e.g. the particular zoo/avery/lab name","",sample_source,"","",m,"",""
 319,sample contact,the person or institute that can be contacted regarding the sample acquisition,"",sample_contact,"","",m,"",""
 320,IUCN Red List,"The IUCN Red List of Threatened Species, is the world's most comprehensive inventory of the global conservation status of biological species. See wikipedia http://en.wikipedia.org/wiki/IUCN_Red_List.  (Extinct (EX) � No known individuals remaining. Extinct in the Wild (EW) � Known only to survive in captivity, or as a naturalized population outside its historic range. Critically Endangered (CR) � Extremely high risk of extinction in the wild. Endangered (EN) � High risk of extinction in the wild. Vulnerable (VU) � High risk of endangerment in the wild. Near Threatened (NT) � Likely to become endangered in the near future. Least Concern (LC) � Lowest risk. Does not qualify for a more at risk category. Widespread and abundant taxa are included in this category. Data Deficient (DD) � Not enough data to make an assessment of its risk of extinction. Not Evaluated (NE) � Has not yet been evaluated against the criteria.)","",red_list,[EX|EW|CR|EN|VU|NT|LC|DD|NE],"",1,"",""
```

Comment:
Is it better to remove the special character `�` from the test data?

Reply:
Not sure what happened but those special characters have now been removed.

Reply:
I removed the odd chars from the live version, so maybe that change got propagated to the test data?